api/src/main/java/org/motechproject/nms/api/web/contract/FrontLineWorkerUser.java

```diff
@@ -7,8 +7,8 @@
     private long currentUsageInPulses;
     private long endOfUsagePromptCounter;
```

Comment:
Right!
I think @larubbio checked that in originally.
I'll put in a ticker to harmonize our response/requests class names.